bundles/org.openhab.binding.daikin/src/main/java/org/openhab/binding/daikin/internal/DaikinHandlerFactory.java

```diff
@@ -34,6 +35,8 @@
 @NonNullByDefault
 public class DaikinHandlerFactory extends BaseThingHandlerFactory {
 
+    private @Nullable DaikinDynamicStateDescriptionProvider stateDescriptionProvider;
```

Comment:
If you use constructor injection you can make it `final` and remove `@Nullable`

Reply:
I removed @NonNullByDefault and @Nullable notations.